HeterogeneousCore/AlpakaMath/interface/deltaPhi.h

```diff
@@ -0,0 +1,36 @@
+#ifndef HeterogeneousCore_AlpakaMath_deltaPhi_h
+#define HeterogeneousCore_AlpakaMath_deltaPhi_h
+
+#include <alpaka/alpaka.hpp>
+
+namespace cms::alpakatools {
+
+  // reduce to [-pi,pi]
+  template <typename TAcc, typename T>
+    requires std::is_floating_point_v<T>
```

Comment:
One alternative that I forgot to mention earlier would be
```suggestion
  template <typename TAcc, std::floating_point T>
```
(just mentioning, not asking to change unless you'd want to)